karyon3-archaius2/src/main/java/com/netflix/karyon/archaius/admin/ArchaiusResource.java

```diff
@@ -16,7 +16,7 @@
 
 @Singleton
 @AdminService(name="props", index="getAllProperties")
-public class ArchaiusResource {
+final class ArchaiusResource {
```

Comment:
Yes, note that these are the v3 resources that may change so we don't actually want people using them directly at this point.  We may make these accessible later.